CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+### 2.0.0.beta1
+
+**Preliminary release of Ruby 3 support**
+
+**Notable changes:**
+ *   Database schema has changed to split job arguments into args and kwargs.
+ *   It's no longer possible to pass keyword arguments whose keys conflict with job option keys.
```

Comment:
@oeoeaio I think this point could do with clarification. With the addition of the `job_options` kwarg and deprecation of including job options straight in the kwargs [in 1.2.0](https://github.com/que-rb/que/blob/master/CHANGELOG.md#120-2022-02-23), there's no conflict anymore, right? Or do we still need to remove the deprecated code path?

Reply:
Resolved - I've dropped support for top-level job options and reworded this.